client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -45,20 +66,24 @@ const StatsNavigation = ( props ) => {
 					{ sectionTitles.year }
 				</NavItem>
 			</NavTabs>
+			{ statsControl }
 			<FollowersCount />
 		</SectionNav>
 	);
 };
 
 StatsNavigation.propTypes = {
+	isJetpack: PropTypes.bool,
+	isWooConnect: PropTypes.bool,
 	section: PropTypes.string.isRequired,
 	slug: PropTypes.string,
+	siteId: PropTypes.number,
 };
 
 const connectComponent = connect(
-	state => {
+	( state, props ) => {
 		return {
-			slug: getSelectedSiteSlug( state )
+			isWooConnect: props.isJetpack && hasActivePlugin( state, props.siteId, 'woocommerce' )
```

Comment:
is there a difference between: 
`props.isJetpack && hasActivePlugin(...)` vs. `hasActivePlugin(...)`?

I would assume that if isJetpack were false then hasActivePlugin would be false as well.